js/charts/clusteredhorizontalbar.js

```diff
@@ -885,75 +884,73 @@ function clusteredhorizontalbar() {
                 .on('mousemove', _handleMouseMoveFn.call(chart, tooltip, _local_svg))
                 .on('mouseout', _handleMouseOutFn.call(chart, tooltip, _local_svg))
                 .on('click', function (d) {
-                    if (COMMON.COMPARABLE_DATA_TYPES.indexOf(_dimensionType[0]) === -1) {
-                        if (!_print) {
-                            if (broadcast != undefined && broadcast.isThresholdAlert) {
-                                var ThresholdViz = {};
-                                ThresholdViz.ID = parentContainer.attr('vizID');
-                                ThresholdViz.measure = d.measure;
-                                ThresholdViz.measureValue = d[d.measure];
-                                ThresholdViz.dimension = d.dimension;
-                                ThresholdViz.dimensionValue = d[d.dimension];
-                                broadcast.ThresholdViz = ThresholdViz;
-                                broadcast.$broadcast('FlairBi:threshold-dialog');
+                    if (!_print) {
+                        if (broadcast != undefined && broadcast.isThresholdAlert) {
+                            var ThresholdViz = {};
+                            ThresholdViz.ID = parentContainer.attr('vizID');
```

Comment:
can u keep a seprate function to set props for threshold

                            var ThresholdViz = {};
                            ThresholdViz.ID = parentContainer.attr('vizID');
                            ThresholdViz.measure = d.measure;
                            ThresholdViz.measureValue = d[d.measure];
                            ThresholdViz.dimension = d.dimension;
                            ThresholdViz.dimensionValue = d[d.dimension];
                            broadcast.ThresholdViz = ThresholdViz;
                            broadcast.$broadcast('FlairBi:threshold-dialog')